test/index.js

```diff
@@ -64,13 +75,13 @@ describe('DDWAF', () => {
   it('should collect an attack and cleanup everything', () => {
     const waf = new DDWAF(rules)
     const context = waf.createContext()
+    const payload = {
+      persistent: {
+        'server.request.headers.no_cookies': 'value_ATTack'
```

Comment:
you have removed data from the payload, why?

```
      x: new Array(4096).fill('x').join(''),
      y: new Array(4097).fill('y').join(''),
      z: new Array(4097).fill('z')
```

Reply:
I've removed it accidentally when I refactored it.